test/proxy.py

```diff
@@ -81,34 +100,194 @@ async def test_call_dapp(contract_factory):
     await sender.send_transaction([(dapp.contract_address, 'set_number', [47])], [signer, guardian])
     assert (await dapp.get_number(account.contract_address).call()).result.number == 47
 
+
+def build_execute_after_upgrade_data(calls: Optional[List[Tuple]] = None) -> List[Tuple]:
```

Comment:
And then use it in lieu of `Tuple`:

```suggestion
def build_execute_after_upgrade_data(calls: Optional[List[Call]] = None) -> List[Tuple]:
```